model/src/channel/message/mod.rs

```diff
@@ -11,6 +13,7 @@ pub use self::{
     flags::MessageFlags, kind::MessageType, reaction::MessageReaction, reference::MessageReference,
 };
 
```

Comment:
should this have a pub use Sticker for consistency or no

Reply:
Added